notus/scanner/cli/parser.py

```diff
@@ -19,19 +19,18 @@
 import logging
 
 from pathlib import Path
+from typing import Any, Dict
 
 from ..config import Config
 from ..__version__ import __version__
 
-DEFAULT_CONFIG_PATH = "/etc/gvm/notus-scanner.conf"
-DEFAULT_PID_PATH = "/run/gvm/notus-scanner.pid"
-DEFAULT_MQTT_BROKER_PORT = 1883
-
 ParserType = argparse.ArgumentParser
 Arguments = argparse.Namespace
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_CONFIG_FILE = "/etc/gvm/notus-scanner.conf"
```

Comment:
Regarding the location of this file it bothers me, that it is different from the location of the ospd.conf file which is located in `~/.conf/ospd.conf`. To keep locations of config files consistent I'd like to either move this one also to `~/.conf/...` or also change the configs of ospd-openvas.
Also as the config file is a TOML file, shouldn't the standard file end with a .toml?

Reply:
On Linux global system wide configs should be placed somewhere in `/etc`. The [XDG](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) standard (a linux desktop standardization group which is mostly driven by GNOME/Fedora/Red Hat) defined that local user configuration should be put somewhere in `XDG_CONFIG_HOME` which defaults to `~/.config`.

IMHO ospd-openvas and also notus-scanner should be run system wide in most cases. Therefore loading the config from `/etc` seems to be better for me. But nevertheless we could first check if a config file in `~/.config` exists and if not load a config from `/etc`. Maybe that's the best approach to support both use cases.

Regarding the `.toml` suffix I think you are right and we should change it. I've just left it as is for easier adjustments in the packages. Will change that.

Reply:
Just as a side note IMHO using TOML in ospd-openvas (and also allow overriding the settings via env variables) would be best too. In general it would be good to standardize our config file usage in some way (for example using `-` for connection words in variable names instead of `_`).